src/rard/research/urls.py

```diff
@@ -649,6 +649,26 @@
                         views.CitingAuthorUpdateView.as_view(),
                         name="update",
                     ),
+                    path(
+                        "<pk>/introduction/",
+                        views.CitingAuthorIntroductionView.as_view(),
+                        name="introduction",
+                    ),
+                    path(
+                        "<pk>/update/introduction/",
+                        views.CitingAuthorUpdateIntroductionView.as_view(),
+                        name="update_introduction",
+                    ),
+                    path(
+                        "<pk>/bibliography/",
+                        views.BibliographySectionView.as_view(),
+                        name="bibliography",
+                    ),
```

Comment:
If using the [passing extra options](https://docs.djangoproject.com/en/5.1/topics/http/urls/#views-extra-options) approach, I think this could work:
```suggestion
                    path(
                        "<pk>/bibliography/",
                        views.BibliographySectionView.as_view(),
                        {"related_model":"citing_author"},
                        name="bibliography",
                    ),
```